release/tools/arm.subr

```diff
@@ -199,7 +199,7 @@ arm_install_base() {
 		> ${CHROOTDIR}/${DESTDIR}/etc/fstab
 	echo "/dev/ufs/rootfs		/		ufs	rw		1	1" \
 		>> ${CHROOTDIR}/${DESTDIR}/etc/fstab
-	echo "/dev/msdosfs/EFI		/boot/efi	msdosfs	rw,noatime	0	0" \
+	echo "/dev/gpt/efiboot0		/boot/efi	msdosfs	rw		2	2" \
```

Comment:
Will this work in the MBR-formatted case?

Reply:
You're absolutely right, it won't work.